integrations/integration_cname.js

```diff
@@ -117,6 +119,7 @@ const commonNames = {
   ...LiveChat,
   ...Shynet,
   ...Woopra,
+  ...QuoraPixel,
```

Comment:
Why are Quora changes in this branch?

Reply:
There are still conflicts in this PR @shrouti1507.

Reply:
Yeah for the ones merged yesterday. Will resolve the conflicts and let you know.

Reply:
@saikumarrs Conflicts are resolved. Please check.